vslib/src/MACsecManager.cpp

```diff
@@ -845,7 +845,10 @@ void MACsecManager::cleanup_macsec_device() const
 
     if (!exec("/sbin/ip macsec show", macsecInfos))
     {
-        SWSS_LOG_THROW("Cannot show MACsec ports");
+        // this is workaround, there was exception thrown here, and it probably
+        // suggest that, ip command is in place, but don't support "macsec"
+        SWSS_LOG_ERROR("Cannot show MACsec ports");
```

Comment:
i think this will block log analyzer, better to change to notice level.

Reply:
what log analyzer ? out internal ? do you propose warning ?

Reply:
the log analyzer in sonic-mgmt repo, it will report error when it does the vsimage check, warning should be fine.